app/assets/stylesheets/pages/home.scss

```diff
@@ -1,3 +1,4 @@
-// Place all the styles related to the Home controller here.
-// They will automatically be included in application.css.
-// You can use Sass (SCSS) here: http://sass-lang.com/
+body h2 {
```

Comment:
this is a very general css rule to have it in the `home.scss`. If this is the rule that we are going to follow on **every** page, we should move this to a more generally specified scss file

Reply:
@sanghun89 Yeah, I see what you mean-- that makes sense. In the next commit, I am just going to get rid of it entirely because we aren't really working with the home page for now (since we decided to change this commit to the "user header" as opposed to just the static home page) so I might just revisit that css when I revisit that home page information. It doesn't change anything that is displayed for now anyways.